src/Application/src/Calculator.cpp

```diff
@@ -12,7 +12,8 @@
 #include <typeinfo>
```

Comment:
Reorder: should have system libraries in alphabetical order a space then your .hs in alphabetical order.

You shouldn't need stdlib.h. Note: You should not use c libraries that end in ".h" like stdlib.h and math.h, there is a version that is correctly wrapped and c++ format. Instead of <math.h> you should use \<cmath\>

```cpp
#include <cmath>
#include <iostream>
//...

#include "CalculatorMessages.h"
//...
```